14-NiM/NiM.cs

```diff
@@ -0,0 +1,62 @@
+﻿// All the variables that will be just in the Game
+ Console.WriteLine("Welcome to Nim!");
+ bool isAiTurn  = false;
+ int matches = 24;
+ int minMatchesToDraw = 1;
+ int maxMatchesToDraw = 3;
+ char matchChar  = '|';
+ Random random = new Random();
+ string matchOutput = new string(matchChar , matches);
+ Console.WriteLine($"{matchOutput} ({matches})");
+ 
+
+//Start of the loop
+ restartFromStart:
+ switch (isAiTurn)
+ {
+  //This is the player segment
+  case false:
+   Console.WriteLine("How many matches do you want to draw?");
+   string? userInput = Console.ReadLine();
+   int y;
+   while (!int.TryParse(userInput, out y) || y < minMatchesToDraw || y > maxMatchesToDraw)
+   {
+    Console.WriteLine("Wrong, give me a number between 1 and 3 and no characters.");
+    userInput = Console.ReadLine();
+   }
+   
+   matches -= y;
+   matchOutput = new string(matchChar , matches);
+   Console.WriteLine($"{matchOutput} ({matches})");
+   
+   if (matches <= 1)
+   {
+    Console.WriteLine("You win!");
+    goto endGame;
+   }
```

Comment:
Rules were misunderstood. You lose if you draw the last match. If there is two left and I draw both, I should lose.